src/qt/walletframe.h

```diff
@@ -31,7 +32,7 @@ class WalletFrame : public QFrame
     Q_OBJECT
 
 public:
-    explicit WalletFrame(const PlatformStyle *platformStyle, BitcoinGUI *_gui = nullptr);
+    WalletFrame(const PlatformStyle* platformStyle, BitcoinGUI* _gui, QWidget* parent);
```

Comment:
Please keep the explicit here. Even though this can likely never happen with three arguments without a default value, it's better to specify it *unless* you want the object to ever be implicitly constructed.

Reply:
Agree.

TIL: from C++11 a [converting constructor](https://en.cppreference.com/w/cpp/language/converting_constructor) can accept _any_ number of parameters, not a single parameter.

Reply:
> Please keep the explicit here.

Done.